gsoc/2022/mentors.md

```diff
@@ -27,3 +27,7 @@ layout: plain
 * Mayank Sharma[Mayank Sharma](mailto:mayank.sharma@cern.ch) CERN
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) imperialcollege
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Felice Pantaleo[Felice Pantaleo](mailto:felice.pantaleo@cern.ch) CERN
```

Comment:
please insert mentors in alphabetical order

Reply:
Done.